test/app/app.rb

```diff
@@ -14,19 +14,20 @@
   ActiveRecord::Base.logger = Logger.new('/dev/stdout')
 end
 
-
+# VERY CAREFUL CHANGING THE LINES OF THIS FILE.  THE TEST WILL BREAK IF YOU ADD LINES
 database_yml = File.join(File.dirname(__FILE__), "..", "database.yml")
 database_yml = database_yml + ".example" unless File.exist?(database_yml)
 
 connection = YAML.load_file(database_yml)
 
-TEST_ENV = ENV['SHIBA_TEST_ENV'] || 'test'
-ActiveRecord::Base.establish_connection(connection[TEST_ENV])
+test_env = ENV['SHIBA_TEST_ENV'] || 'test'
+test_env = "test_#{test_env}" unless test_env.start_with?("test")
+ActiveRecord::Base.establish_connection(connection[test_env])
 
 org = Organization.create!(name: 'test')
 org.users.create!(email: 'squirrel@example.com')
-
-user = User.find_by(email: 'squirrel@example.com') # bumpity
+users = User.where(email: 'squirrel@example.com').to_a # bumpity
```

Comment:
Table Scan: The database reads 100% (100000) of the of the rows in **users**, skipping any indexes.
Estimated query time: 0.02s